packages/app/src/composables/useAddress.ts

```diff
@@ -62,13 +62,16 @@ type ContractVerificationRequest = {
   optimizationUsed: boolean;
 };
 
+export const VERIFICATION_PROBLEM_INCORRECT_METADATA = "incorrectMetadata";
```

Comment:
Not used anywhere right now, but thought that it's still good to add in case it will ever be needed (e.g. when more problems will be added and we will handle them differently in UI).